server/routes/message-create.js

```diff
@@ -63,7 +63,7 @@ module.exports = [
         return boom.internal('Problem creating message.', res)
       }
 
-      return h.redirect('/messages')
+      return h.redirect(`/message-view/${message.id}`)
```

Comment:
Display view of the message after creating it rather than just the list of messages.